.github/workflows/test-report.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Test Report'
+on:
+  workflow_run:
+    workflows: ['Main pipeline']
+    branches: [main, MigrateToMainn]
```

Comment:
`MigrateToMainn` probably isn't necessary here (if you do want to include it, there's a typo)

Reply:
You are correct